From mikey@neuling.org  Thu Aug  5 15:51:58 2010
From: Michael Neuling <mikey@neuling.org>
Date: Wed, 04 Aug 2010 08:32:13 +1000
Subject: CIFS: Fix compile error with __init in cifs_init_dns_resolver() definition
Subject: [stable] [PATCH 2.6.32/stable] CIFS: Fix compile error with __init in cifs_init_dns_resolver() definition
To: David Howells <dhowells@redhat.com>
Cc: linux-cifs@vger.kernel.org, stable@kernel.org, Jeff Layton <jlayton@redhat.com>, linux-kernel@vger.kernel.org
Message-ID: <20749.1280874733@neuling.org>

From: Michael Neuling <mikey@neuling.org>

An allmodconfig compile on ppc64 with 2.6.32.17 currently gives this error

fs/cifs/dns_resolve.h:27: error: expected '=', ',', ';', 'asm' or '__attribute__' before 'cifs_init_dns_resolver'

This adds the correct header file to fix this.

Signed-off-by: Michael Neuling <mikey@neuling.org>
Cc: David Howells <dhowells@redhat.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 fs/cifs/dns_resolve.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/fs/cifs/dns_resolve.h
+++ b/fs/cifs/dns_resolve.h
@@ -24,6 +24,8 @@
 #define _DNS_RESOLVE_H
 
 #ifdef __KERNEL__
+#include <linux/module.h>
+
 extern int __init cifs_init_dns_resolver(void);
 extern void cifs_exit_dns_resolver(void);
 extern int dns_resolve_server_name_to_ip(const char *unc, char **ip_addr);
